### DISCUSSION TITLE
custom snippet languages

### DISCUSSION THREAD

**Discussion by tizu69** [scalar/scalar#3362] [Ideas]
titlte should be self explanitory

**Comment by marclave** [comment]
we support this with [x-code-samples](https://github.com/scalar/scalar/blob/main/documentation/openapi.md#x-codesamples)

is there something more you would want? :)

**Reply by tizu69** [reply]
ah, I see. However, I'm still a bit confused. is there a way to add a global code sample, instead of having to define it per route? what abt removing existing samples?
